docker-compose.yml

```diff
@@ -2,8 +2,9 @@ version: '3.8'
 
 services:
   px4-gazebo-headless:
+    platform: linux/amd64
     # image: jonasvautherin/px4-gazebo-headless:latest
-    image: jonasvautherin/px4-gazebo-headless:1.11.0
+    image: jonasvautherin/px4-gazebo-headless:1.12.3
```

Comment:
Have you tested 1.12? I think that the last time I tested it, there was I problem. Not sure if it was MAVSDK related or something else.

Reply:
let s revert then 😁